### ISSUE TITLE
Create Written Workflow Tutorials

### ISSUE THREAD

**Issue by OpenBagTwo** [OpenBagTwo/EnderChest#14]
**GIVEN** a user has read the intro, installed the package and has gained basic familiarity with the interface via `enderchest --help`
**WHEN** they go to the docs page and click the link for "Tutorials"
**THEN** they're taken to a page or pages with written tutorials, complete with screenshots, that walk them through expected workflows of living with and using EnderChest
**SO** that they don't have to figure out their own workflows in order to actually use the package

### Example Workflows
There are two (and a half) example workflows that should be covered:
- The original workflow where all the real files are stored inside the EnderChest folder
- The "Chest Monster" workflow where all the real files are stored elsewhere, and EnderChest just contains links
- My current hybrid workflow, where mods and configs are stored in EnderChest, with large assets or like worlds or in-development assets like resource packs stored and synced separately

### Steps to Cover
1. Finding your minecraft folders when dealing with the official launcher, PrismLauncher and one other (Curseforge? Lunar? GD?)
2. Creating your "minecraft root" (linking instances)
3. Initial `craft` (with and without a config file)
4. Moving assets into the EnderChest folder
5. First `place` (and testing everything)
6. Setting up and using version control
7. Modifying `open.sh` and `close.sh`
8. First syncs (`open` / `close`)
9. Maintenance: updating assets
10. Maintenance: onboarding a new installation
11. Best practices and antipatterns

### Out of Scope
- Server management

### Implementation Note

Not blocked by #11 as this could very easily be hosted in the wiki

**Comment by OpenBagTwo** [comment]
Just jotting down some quick notes on best practices and antipatterns:

- **DO** use relative links
- **DO** store your _actual instance data_ inside your minecraft root (and then put symlinks from the launchers pointing _into_ the minecraft root)
  - include a note on managing sandbox permissions
- **AVOID** linking to stuff outside of the minecraft root
- For configs, favor "local-only" over "client-only"
- **DO** use `--dry-run` and `--verbose` whenever futzing with `rsync`
  - but also know that a successful dry run will not protect you from an rsync gone wrong

**Comment by OpenBagTwo** [comment]
Much of this is now covered by [the docs](https://openbagtwo.github.io/EnderChest/dev/usage/), but I'm leaving this open until I've dogfooded the new rewrite for a while and fleshed out [this](https://openbagtwo.github.io/EnderChest/dev/suggestions/)

**Comment by OpenBagTwo** [comment]
... or until I've spun off a new issue
